salt/modules/heat.py

```diff
@@ -102,8 +102,8 @@ def _construct_yaml_str(self, node):
     Construct for yaml
     '''
     return self.construct_scalar(node)
-YamlLoader.add_constructor(u'tag:yaml.org,2002:str',
-                           _construct_yaml_str)
+# YamlLoader.add_constructor(u'tag:yaml.org,2002:str',
```

Comment:
We don't like to have dead code in our code base. Instead of commenting these out, please just remove these lines.